tests/fields/fields.go

```diff
@@ -17,13 +17,13 @@
 package main
 
 import (
+	"code.google.com/p/goauth2/oauth"
```

Comment:
@gmlewis I'm able to make it look like this with no issues to `gofmt`:

```
    "encoding/json"
    "flag"
    "fmt"
    "os"
    "reflect"
    "strings"

    "code.google.com/p/goauth2/oauth"
    "github.com/google/go-github/github"
```

just to be clear, is this what you mean?